src/components/Carousel/Carousel.js

```diff
@@ -47,61 +44,85 @@ class Carousel extends Component {
   onResize() {
     this.resizeTheCarousel();
     this.checkNumOfSlidesToScroll();
+    this.getSlidesCount();
   }
 
-  resizeTheCarousel() {
-    const { widthOfSlide } = this.state;
-    let carouselWidth = '0';
-
-    if (window.innerWidth > 1280) {
-      carouselWidth = `${widthOfSlide * 5}px`;
-    } else if (window.innerWidth > 1024) {
-      carouselWidth = `${widthOfSlide * 4}px`;
-    } else if (window.innerWidth > 768) {
-      carouselWidth = `${widthOfSlide * 3}px`;
-    } else {
-      carouselWidth = `${widthOfSlide * 2}px`;
+  getSlidesCount = () => {
+    const sizeValues = {
```

Comment:
that constant could be moved under imports. Another way you'll recreate this constant each time function get's called.

Reply:
done